react-app/src/components/ResultSidebar.js

```diff
@@ -15,6 +15,37 @@ function ResultSidebar(props) {
     // Save the results permanently somehow
   }
 
+  function getLoggerStats() {
+    let errors = 0;
+    let warnings = 0;
+    let debugs = 0;
+    let total = 0;
+
+    props.loggedMessages.forEach((log) => {
+      if (log.level === 'error') {
+        errors += 1;
+      }
+      if (log.level === 'warn') {
+        warnings += 1;
+      }
+      if (log.level === 'debug') {
+        debugs += 1;
+      }
+      total += 1;
+    });
+
+    return (
+      <Accordion.Body>
+        <p key={0}>Errors: {errors}</p>
+        <p key={1}>Warnings: {warnings}</p>
+        <p key={2}>Debug Messages: {debugs}</p>
```

Comment:
I don't think we need to report the number of debug messages. The debug messages just provide additional detail, but the number of them isn't really indicative of the number of problems that will require attention, like the error and warn messages.